addons/collapse-footer/userstyle.css

```
@@ -0,0 +1,31 @@
+.sa-collapse-footer, /* scratch-www */
+.sa-collapse-footer #content /* scratchr2 */ {
+  padding-bottom: 31px;
+}
+
+.sa-collapse-footer #footer {
+  position: fixed;
+  top: initial;
+  bottom: 0;
+  width: 100%;
+  border-top: 1px solid #d9d9d9;
+  height: 32px !important;
+  box-sizing: border-box;
+  padding-top: 4px !important;
+  z-index: 20;
+  transition-duration: 0.5s;
+}
+.sa-collapse-footer #footer .lists dl {
+  margin-top: 0;
+}
+.sa-collapse-footer #footer:hover {
+  height: var(--footer-hover-height) !important;
+}
```

The footer might look a little more pleasing if the top margin was consistent with the vanilla footer, only when the footer is expanded:
```suggestion
.sa-collapse-footer #footer .lists dl {
  margin-top: 0;
  transition-duration: 0.5s;
}
.sa-collapse-footer #footer:hover {
  height: var(--footer-hover-height) !important;
}
.sa-collapse-footer #footer:hover .lists dl {
  margin-top: 12px;
}
```

---

Wait, that doesn't cover `scratchr2`...

---

I didn't bother changing the CSS from the old version except for the selectors.